### ISSUE TITLE
About the loss when training

### ISSUE THREAD

**Issue by jinqin-99** [HqiTao/CT-crackseg#2]
hello, you have done good job! BUT When I am running your code, I have some problems. I found that **the loss** do not go down.

![image](https://github.com/HqiTao/CT-crackseg/assets/60920383/d1975086-381c-479b-92e7-59c81d2277bf)
**This is my config file.**

![image](https://github.com/HqiTao/CT-crackseg/assets/60920383/16f222f0-6942-40e2-8922-fbc9fde35104)
**This is the part of loss_log.txt**

Please tell me what I should do.Thank you very much!

**Comment by HqiTao** [comment]
You can try increasing the learning rate.

In your configuration file, you increased the batch size without adjusting the learning rate. Empirically, when batch size is raised, the learning rate should be increased proportionately.

**Comment by ABCnutter** [comment]
The CT-CrackSeg code I assume is written and run on Windows. In the train_crack.py file, the data read in lines 30-34 is fine on Windows, but if you run it in Ubuntu, it reads incorrectly because Ubuntu reads the filenames out of order like that. You can import "from natsort import natsorted", and then use natsorted to wrap the array of filenames read from lines 30-34, which solves the Loss problem.

**Comment by ABCnutter** [comment]
The CT-CrackSeg code I assume is written and run on Windows. In the train_crack.py file, the data reading of 30-34 is fine on Windows, but if you run it in Ubuntu, it reads incorrectly. This is because Ubuntu reads the filenames out of order. You can import "from natsort import natsorted", and then use natsorted to wrap the array of filenames read from lines 30-34, which solves the Loss problem.

**Comment by jinqin-99** [comment]
> The CT-CrackSeg code I assume is written and run on Windows. In the train_crack.py file, the data reading of 30-34 is fine on Windows, but if you run it in Ubuntu, it reads incorrectly. This is because Ubuntu reads the filenames out of order. You can import "from natsort import natsorted", and then use natsorted to wrap the array of filenames read from lines 30-34, which solves the Loss problem.

You're right. It's the data loaded in Linux that gets out of order. Your solution is amazing! Thank you for your reply
